code/game/objects/structures/watercloset.dm

```diff
@@ -47,12 +47,62 @@
 	open = !open
 	update_icon()
 
-
 /obj/structure/toilet/update_icon()
 	icon_state = "toilet[open][cistern]"
-
+	if(!anchored)
+		pixel_x = 0
+		pixel_y = 0
+		layer = OBJ_LAYER
+	else
+		if(dir == SOUTH)
+			pixel_x = 0
+			pixel_y = 8
+		if(dir == NORTH)
+			pixel_x = 0
+			pixel_y = -8
+			layer = FLY_LAYER
 
 /obj/structure/toilet/attackby(obj/item/I, mob/living/user, params)
+	if(iswrench(I))
+		if(anchored)
```

Comment:
That is quite frankly a much better way to handle it that I didn't think of at the time. Though, in complete honesty, I also didn't think I needed to make various exceptions when I began it either.